src/bindings/crypto_sign.h

```diff
@@ -33,10 +33,12 @@ int crypto_sign_open(unsigned char *m,  unsigned long long *mlen,
                const unsigned char *sm, unsigned long long smlen,
                const unsigned char *pk);
 
-
 int crypto_sign_ed25519_sk_to_pk(unsigned char *pk,
```

Comment:
If you verified the direct slicing to be faster than calling libsodium's crypto_sign_ed25519_sk_to_pk(), you should also remove the now unused binding here.

Reply:
Done. Thanks for being patient as I'm learning my way around the codebase :)